recipes/qhull/all/conanfile.py

```diff
@@ -42,14 +42,10 @@ def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
 
-    @property
-    def _is_v8_1(self):
-        return self.version == "8.1-alpha4" or Version(self.version) >= "8.1"
-
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
-        if self._is_v8_1:
+        if Version(self.version) >= "8.1-alpha4":
```

Comment:
```suggestion
        if Version(self.version, True) >= "8.1":
```
@perseoGI to make it work with both the pre-release and a final `8.1` release.